ui/page/dcrdex/market.go

```diff
@@ -1776,14 +1794,13 @@ func (pg *DEXMarketPage) refreshOrders() {
 	pg.orders = nil
 	for i := range orders {
 		ord := &clickableOrder{Order: orders[i]}
-		if ord.Status == order.OrderStatusExecuted && anyMatchActive(ord.Matches) != pg.openOrdersDisplayed /* display active orders on open order view */ {
+		hasActiveMatch := anyMatchActive(ord.Matches)
+		if ord.Status == order.OrderStatusExecuted && hasActiveMatch != pg.openOrdersDisplayed /* display active orders on open order view */ {
 			continue // skip order
 		}
 
-		if pg.openOrdersDisplayed && !ord.Cancelling {
-			btn := pg.Theme.DangerButton(values.String(values.StrCancel))
-			btn.Margin = layout.Inset{}
-			ord.cancelBtn = &btn
+		if pg.openOrdersDisplayed && !ord.Cancelling && !hasActiveMatch {
+			ord.cancelBtn = pg.Theme.NewClickable(false)
```

Comment:
You can cancel even if there's an active match; if there's an unmatched quantity. I think the correct check would be `if ord.Status != order.OrderStatusExecuted`.